### ISSUE TITLE
Scribefire post not published

### ISSUE THREAD

**Issue by GoogleCodeExporter** [chrsmith/scribefire-chrome#828]
```
What's the problem?

I publish the post to my blog but it doesn't appear there.

What browser are you using?

Firefox 28.0

What version of ScribeFire are you running?

4.2.4

```

-----
Original issue reported on code.google.com by `begoger...@chemistswithoutborders.org` on 2 Apr 2014 at 7:03

**Comment by GoogleCodeExporter** [comment]
```
Which Blogging Type are you using?
```

-----
Original comment by `scribefi...@gmail.com` on 4 Apr 2014 at 6:46

**Comment by GoogleCodeExporter** [comment]
```
Thanks. I use Blogger: chemistswithoutborders.blogspot.com.

Peace,

Bego
```

-----
Original comment by `begoger...@chemistswithoutborders.org` on 4 Apr 2014 at 10:08

**Comment by GoogleCodeExporter** [comment]
```
Fixed for blogger blogspot in version 4.3.1
```

-----
Original comment by `scribefi...@gmail.com` on 14 Apr 2014 at 6:57
* Changed state: **Fixed**
